frontend/app/electron/main/tray-manager.ts

```diff
@@ -52,10 +67,18 @@ export class TrayManager {
     ]);
   }
 
-  update({ currency, delta, percentage, up, period, netWorth }: TrayUpdate) {
+  update(tray?: TrayUpdate) {
     if (!this.tray)
       return;
 
+    const trayData = tray ?? this.trayData;
+
+    if (!trayData)
+      return;
+
+    const { currency, delta, percentage, up, period, netWorth } = trayData;
+    this.trayData = trayData;
+
```

Comment:
So I noticed an interesting bug on windows in a new account. It should be probably easy to replicate in a new account on macOS too.

`up` is undefined because it's a new account but networth is not empty. 

Do you think we can do a small change here? if `netWorth` is defined/non-empty then show `Net worth ${netWorth} ${currency}.` Otherwise show the `rotki is running` text

Reply:
This is for line 85 below